docs/insights-archive-sample/config/virtualmachineinstances/default/fedora-r2nf0eocvxbkmqjy.json

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
I think the resource will probably not be created in `default` namespace. It would be better to try to reflect the real "deployment" in our sample archive.

Reply:
Any suggestions? What do you mean by "real deployment"?

Reply:
In my case the new VMI resource was created in `openshift-cnv` namespace.

Reply:
Yeah, it can be created in any namespace, actually... There is no limitation, it depends on the project that is active.